### PR TITLE
Adding support for subPath

### PR DESCRIPTION
**Pull request by gandazgul**

### CODE REVIEW

**Comment by gandazgul** [comment]
@itzg thanks, sorry it took me so long to come back to this. I bumped the chart version.

File: charts/minecraft/templates/deployment.yaml [hunk 1]
```diff
@@ -239,6 +239,9 @@ spec:
         volumeMounts:
         - name: datadir
           mountPath: /data
+          {{- if (and .Values.persistence.dataDir.enabled .Values.persistence.dataDir.subPath) }}
+          subPath: {{ .Values.persistence.dataDir.subPath }}
```
Comment (billimek) [inline charts/minecraft/templates/deployment.yaml:243]: Should we also add a placeholder in the `values.yaml` file with an example (commented) of what this setting would be.

Generally speaking, if we reference a value, it's a good practice to have an example in the values.yaml file.

Comment (itzg) [reply charts/minecraft/templates/deployment.yaml:243]: Good point. Can you add that @gandazgul ?

Comment (gandazgul) [reply charts/minecraft/templates/deployment.yaml:342]: This is done

**Review by itzg — APPROVED** [review]

**Comment by carlos-ravelo-ask** [comment]
thanks!

**Pull request merged**